Emby.Naming/AudioBook/AudioBookListResolver.cs

```diff
@@ -50,20 +48,9 @@ public IEnumerable<AudioBookInfo> Resolve(IEnumerable<FileSystemMetadata> files)
                     Files = stackFiles,
                     Name = stack.Name
                 };
-                list.Add(info);
-            }
-
-            // Whatever files are left, just add them
-            /*list.AddRange(remainingFiles.Select(i => new AudioBookInfo
-            {
-                Files = new List<AudioBookFileInfo> { i },
-                Name = i.,
-                Year = i.Year
-            }));*/
-
-            var orderedList = list.OrderBy(i => i.Name);
```

Comment:
this drops ordering, not sure we really want that

Reply:
The tests pass 🤷‍♂️

Reply:
I'm just on the wary side since ordering broke real clients before...